src/System.Text.Utf8String/System/Text/Utf8String.cs

```diff
@@ -587,6 +587,43 @@ public Utf8String TrimStart(byte[] trimCodeUnits)
             throw new NotImplementedException();
         }
 
+        public Utf8String TrimStart(Utf8String prefix)
```

Comment:
This implementation removes any "char" in prefix from the start of the original string. There is an alternative implementation that removes the prefix if found at the beginning of the string, i.e.
a) "abad".TrimStart("ab") => "ad"
b) "abad".TrimStart("ab") => "d"

Which of these two should we implement? It might be worth asking the person who requested this API.

If the they wanted the currently implemented semantics (case b), then the parameter should probably not be called prefix.